### PR TITLE
[Azure Search] Fix broken Search tests

### PR DESCRIPTION
**Pull request by bitsforthought**
FYI @shahabhijeet @chidozieononiwu @weshaggard 

This is just to unblock Search tests until the source tree migration is done.

### CODE REVIEW

**Review by weshaggard — APPROVED** [review]

**Comment by brjohnstmsft** [comment]
@weshaggard Apparently you don't have merge permissions?

**Comment by weshaggard** [comment]
@brjohnstmsft, I have merge permissions, don't you have merge permissions?

**Comment by brjohnstmsft** [comment]
@weshaggard I was confused because even though you had signed off, there was still a red banner saying that someone with merge permissions needed to sign off. Is this because you disabled the merge button for everybody, and like you said, we all have to use the command line to merge?

I don’t know if I can push to master or not, but in my opinion service teams shouldn’t have that power. :-)

**Comment by weshaggard** [comment]
We do require 1 reviewer and all status checks to be green before you can merge but otherwise anyone with write access should be able to merge. While I disabled the "Create a merge commit" option but the "Squash and merge" but should be available. I'm not sure why it was still showing red for you, could have been just a github refresh issue.

> I don’t know if I can push to master or not, but in my opinion service teams shouldn’t have that power. :-)

People with write access can push directly to master (note no one can force push as that is disabled), however in general most folks should not be pushing directly to master and those should come in via merging PR's. For feature branches it is much more acceptable for folks to directly push to.

**Pull request merged**
